ci/check_conda_nightly_env.py

```diff
@@ -15,6 +15,7 @@
     "cubinlinker",
     "pynvjitlink",
     "rapids-dask-dependency",
+    "rapids-logger", # Also not built every night
```

Comment:
Style checks want this:
```suggestion
    "rapids-logger",  # Also not built every night
```